### PR TITLE
converted tests to use flow from publisher & subscriber instead of killswitch

### PR DESCRIPTION
**Pull request by hochgi**
Tests are now deterministic, and not affected by race conditions.
Ref: #108

### CODE REVIEW

**Comment by johanandren** [comment]
Unrelated test fail it seems. I'll retrigger

**Comment by hochgi** [comment]
@raboof it's not "instabilities", as there is no guarantees that killSwitch cancel upstream or downstream first, nor that cancellations propagate to immediately.
behavior changed, but it was never guaranteed, as @johanandren said: https://github.com/akka/akka-stream-contrib/issues/108#issuecomment-317651357
current PR fixes the issue by not using killSwitch which is hard to reason with in the context of these tests.

Bottom line: **NOT a bug in killSwitch!**

File: contrib/src/test/resources/application.conf [hunk 1]
```diff
@@ -0,0 +1,8 @@
+akka {
+  stream {
+    contrib {
+      # Setting timeout to wait only 100 millis instead of 1 minute for tests
+      unfold-flow-timeout = 300ms
```
[the remark this answers is not in the document]
Comment (hochgi) [reply contrib/src/test/resources/application.conf]: Changed!
